File: FrontApp/src/app/Modelo/Factura.ts
 
 export class Factura {
    id:number;
    numeroFactura:number;
    fecha:string;
    tipodePago:string;
    documentoCliente:string;
    nombreCliente:string;
    subtotal:number;
    descuento:number;
    iva:number;
    totalDescuento:number;
    totalImpuesto:number;
    total:number;
    constructor(id:number,numeroFactura:number,fecha:string,tipodePago:string,documentoCliente:string,
    
    nombreCliente:string,
    subtotal:number,
    descuento:number,
    iva:number,
    totalDescuento:number,
    totalImpuesto:number,
    total:number      )
{
 this.id=id;
 this.numeroFactura=numeroFactura;
 this.fecha=fecha;
 this.fecha=fecha;
 this.tipodePago=tipodePago;
 this.documentoCliente=documentoCliente;
 this.nombreCliente=nombreCliente;
 this.subtotal=subtotal;
 this.descuento=descuento;
 this.iva=iva;
 this.totalDescuento=totalDescuento;
 this.totalImpuesto=totalImpuesto;
 this.total=total;
}
}

File: Proyecto/Domain/Model/Factura.cs
namespace Proyecto.Domain.Model
{
    public class Factura
    {
        private int numeroFactura = 0;
        private string fecha;
        private string tipodePago =null;
        private string documentoCliente=null;
        private string nombreCliente = null;
        private decimal subtotal = 0;
        private decimal descuento = 0;
        private decimal iva = 0;
        private decimal totalDescuento = 0;
        private decimal totalImpuesto = 0;
        private decimal total = 0;
       
    
        public int Id { get; set; }

        public int NumeroFactura { get { return numeroFactura; } set { numeroFactura = value; } }

        public string Fecha { get { return fecha; } set { fecha = value; } }
        public string TipodePago { get { return tipodePago; } set { tipodePago = value; } }
        public string DocumentoCliente { get { return documentoCliente; } set { documentoCliente = value; } }
        public string NombreCliente { get { return nombreCliente; } set { nombreCliente = value; } }
        public decimal Subtotal { get { return subtotal; } set { subtotal = value; } }
        public decimal Descuento { get { return descuento; } set { descuento = value; } }
        public decimal Iva { get { return iva; } set { iva = value; } }

        public decimal TotalDescuento { get { return totalDescuento; } set { totalDescuento = value; } }
        public decimal TotalImpuesto { get { return totalImpuesto; } set { totalImpuesto = value; } }
        public decimal Total { get { return total; } set { total = value; } }

        

    }
}
